include/wx/event.h

```diff
@@ -4274,8 +4274,10 @@ typedef void (wxEvtHandler::*wxPressAndTapEventFunction)(wxPressAndTapEvent&);
         static const wxEventTableEntry sm_eventTableEntries[];          \
     protected:                                                          \
         wxCLANG_WARNING_SUPPRESS(inconsistent-missing-override)         \
```

Comment:
The idea was to make this unnecessary.

Reply:
Clang 11 will get suggess-override warning, too (in version 11) and it will have two separate warnings for this.
wxCLANG_WARNING_SUPPRESS uses __has_warning to check if a warning is available, so it works.

I suppose, I can add the inconsistent-missing-override, but this wasn't my goal.

Reply:
Could you please still add it, maybe only for clang < 11 if the new warning will replace the old one in the latter? It's definitely not ideal to have to use 2 different macros for disabling essentially the same warning, just called differently. TIA!

Reply:
OK. Done in b14d085